ConfigurationSystem/Client/SyncPlugins/DummySyncPlugin.py

```diff
@@ -0,0 +1,38 @@
+"""
+DummySyncPlugin
+"""
+
+
+class DummySyncPlugin(object):
+  """ Dummy Synchronization plugin that does nothing.
+      It is used to document what are the requirements for a real Sync plugin.
+
+      Such plugins are meant to validate user's information about to be added to the CS,
+      or to complete it with various sources.
+      They are called by the
+      :py:meth:`DIRAC.ConfigurationSystem.Client.VOMS2CSSynchronizer.VOMS2CSSynchronizer.syncCSWithVOMS`
+  """
+
+  def __init__(self):
+    """ The constructor does not receive any argument.
+        Note that the plugin is instanciated only once, so this is
+        a good place to do global initialization.
+    """
+    pass
+
+  def verifyAndExtendUserInfo(self, username, userDict):
```

Comment:
If this is a stub for plugins, I would simply call this

```suggestion
  def execute(self, *args):
```

Reply:
The method needs to have this name and signature as it's called by `VOMS2CSSynchronizer.py`:
https://github.com/DIRACGrid/DIRAC/blob/7a1953eed680f999f51ffabd61e05078a6af293b/ConfigurationSystem/Client/VOMS2CSSynchronizer.py#L290-L295

Reply:
I must admit I don't like such generic names, especially when we know exactly what it is meant for. It's not like the RMS operations, where there indeed `execute` is as precise as you can get. To be honest, I was at first even considering having two methods, one `verify` and one `extend`, but then realized that it's hard to decouple sometimes (LHCb use case for example)

Reply:
I see, but I suggest to change to a more generic name.

Reply:
if they are "plugins" then they should be generic names IMHO.

Reply:
I agree with Chris here, I'd much rather be able to grep for `verifyAndExtendUserInfo` to see where it is used and implemented rather than guessing. Is there any benefit to the generic name?

Reply:
The benefit is that if we have another plugin where we don't "verify and extend the users info" but rather we, e.g., "drop unused info" or we "filter users", or "whatever", then the plugin would need to implement those functionalities in the `verifyAndExtendUserInfo` method anyway. Plugins interface need to be generic.

Reply:
The only possible things to do with this interface as:
* Reject a new user
* Modify the entry which is about to be created

Of course you could decided to edit the CS directly as a side effect of calling `verifyAndExtendUserInfo` but that would be an abuse of the interface and prone to breaking at any moment. I can see the argument that "extend" is the wrong name so how about then how about renaming it to `verifyAndUpdateUserInfo`?

Reply:
they are generic in the sense that we don't say what/how to verify/update.

Reply:
I'm happy with `verifyAndUpdateUserInfo`